src/app/ngrx/reducers/router.serializer.ts

```diff
@@ -75,7 +75,7 @@ export class CustomSerializer implements RouterStateSerializer<RouterModel> {
         router.endDate = endDate;
       }
       if (params['episodes']) {
-        router.episodeIds = params['episodes'].split(',').map(stringValue => +stringValue);
+        router.episodeIds = params['episodes'].split(',').map(stringValue => +stringValue).filter(id => id > 0);
```

Comment:
Empty string was resulting in a single entry of zero, episode Ids are > 0. Saw a bug, so made a quick fix.